### DISCUSSION TITLE
How do i put a link into the pre-made head tag?

### DISCUSSION THREAD

**Discussion by rn3r** [idom-team/idom#833] [Question]
I dont want to create a div that holds the head that holds the css, i would like a way for me to automatically put it in the head so everything loads normally.

**Comment by Archmonger** [comment]
Related to comments in https://github.com/idom-team/idom/pull/832

**Reply by rn3r** [reply]
i just looked through that and it doesnt seem to be what im looking for. I'm trying to preset the head to contain a link tag to my css

**Reply by Archmonger** [reply]
TLDR: Populating the HTML `<head>` is currently not possible within IDOM's python interface.

It is however, possible using the [javascript components interface](https://idom-docs.herokuapp.com/docs/guides/escape-hatches/javascript-components.html#custom-javascript-components) and using [`react-helmet`](https://github.com/nfl/react-helmet).

**Reply by Archmonger** [reply]
Also, related to https://github.com/idom-team/idom/issues/725

**Comment by rmorshea** [comment]
This has been requested a number of times. While certain things can be done by using `script()` to change the page title after the initial load, this is sub-optimal. I've begun working on a solution to this here: https://github.com/idom-team/idom/pull/835

The approach will be to allow users to configure the `<head/>` element with a `head` option for each of the built-in backend at startup. The interface will look something like:

```python
from starlette import Starlette
from idom import component, html
from idom.backend.starlette import configure, Options

app = Starlette()

@component
def example_app():
    ...

configure(app, example_app, Options(head=html.head(...)))
```
